packages/ui-link/src/Link/styles.ts

```diff
@@ -80,52 +144,46 @@ const generateStyle = (
     fontWeight: componentTheme.fontWeight,
     transition: 'outline-color 0.2s',
```

Comment:
You can remove this `transition` prop here, its now handled by `calcFocusOutlineStyles`